src/components/todo-list/TodoListWrapper.tsx

```diff
@@ -0,0 +1,11 @@
+import { TodoList } from '.';
+
+const TodoListWrapper = () => {
+  return (
+    <>
+      <TodoList />
+    </>
+  );
+};
```

Comment:
??? No make sense extra component. Why do you need it?

Reply:
You shown me a Wrapper as a trick to handle error or loading state. And I decided it's a good point. Removed.